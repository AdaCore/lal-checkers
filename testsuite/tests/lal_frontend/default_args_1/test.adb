procedure Test is
   function F(X : Integer := 0) return Integer is
   begin
      return X;
   end F;

   R : Integer := F;
begin
   null;
end Ex1;
