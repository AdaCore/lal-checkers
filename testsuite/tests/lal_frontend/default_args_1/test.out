Program:
  R#1 = <SubpBody ["F"] test.adb:2:4-5:10>(0)
  end:
